pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go

```diff
@@ -32,18 +32,17 @@ const (
 var files = []string{
 	"crds/submariner.io_submariners_crd.yaml",
 	"crds/submariner.io_servicediscoveries_crd.yaml",
+	"crds/lighthouse.submariner.io_serviceexports_crd.yaml",
+	"crds/lighthouse.submariner.io_serviceimports_crd.yaml",
+	"cluster_role.yaml",
+	"cluster_role_binding.yaml",
+	"globalnet_cluster_role.yaml",
```

Comment:
can we prefix those files with submariner_... (the ones which were in the submariner dir)?

submariner directory meant (handled by submariner, not the operator)
lighthouse directory meant (handled by lighthouse, not the operator)

We just have them here to be able to install the CRDs and do the deploy.

Reply:
I changed the names because I removed the directory.
do you mean the operator should be aware of these definitions?
I made this change so that when generating the csv those definitions would be included as well.